politeia_sync.go

```diff
@@ -0,0 +1,293 @@
+package dcrlibwallet
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/asdine/storm"
+	"github.com/asdine/storm/q"
+	www "github.com/decred/politeia/politeiawww/api/www/v1"
+)
+
+type ProposalCategory int
+
+const (
+	AllProposals = iota + 1
+	PreProposals
+	ActiveProposals
+	ApprovedProposals
+	RejectedProposals
+	AbandonedProposals
+)
```

Comment:
This should be declared in `politeia.go` but here's how to make these constants available for mobile.
```suggestion
const (
      // ProposalCategory
	ProposalCategoryAllProposals       int32 = 1
	ProposalCategoryPreProposals             = 2
	ProposalCategoryActiveProposals          = 3
	ProposalCategoryApprovedProposals        = 4
	ProposalCategoryRejectedProposals        = 5
	ProposalCategoryAbandonedProposals       = 5
)
```
Make sure it is `int32`, using int is unnecessary in this case.